src/main/routes.ts

```diff
@@ -10,4 +10,6 @@ export default function(app: Application): void {
   app.get('/service-choose-action', app.locals.container.cradle.chooseActionController.get);
   app.post('/service-choose-action', app.locals.container.cradle.chooseActionController.post);
   app.get('/courts/:slug', app.locals.container.cradle.courtDetailsController.get);
+  app.get('/individual-location-pages/courts/:slug', app.locals.container.cradle.courtDetailsController.get);
```

Comment:
Isn't this the same as the one above? I think we decided not to use the urls from the prototype and make our own so we settled on /courts/:slug

Reply:
I think I included it by mistake